llvm/tools/llvm-dwarfdump/llvm-dwarfdump.cpp

```diff
@@ -834,8 +842,10 @@ int main(int argc, char **argv) {
                           "-verbose is currently not supported";
     return 1;
   }
-  if (!Verify && ErrorDetails != Unspecified)
-    WithColor::warning() << "-error-detail has no affect without -verify";
+  // -error-detail and -json-summary-file both imply -verify
+  if (ErrorDetails != Unspecified || !JsonSummaryFile.empty()) {
+    Verify = true;
```

Comment:
Ah, I see other comments talked about this. Well, if the others prefer implicit flags, I count myself outvoted 😅

Reply:
You've already typed out either 'json-summary-file' or 'error-detail=...' which both seem pretty darned explicit to me. (Just thinking about forcing more typing with no difference in the end result makes my wrists twinge.) But if folks feel strongly, I can change it back...

Reply:
> You've already typed out either 'json-summary-file'
> seem pretty darned explicit to me

Maybe this is a language thing, but what about "json/summary/files" is "darned explicit" that we want to *verify*? 
IMO this reminds us a lot more about statistics than errors

Reply:
... which I now realize is the point Adrian was trying to make

Reply:
I'm still happy to change it to literally *anything* someone who uses this thing regularly wants me to change it to. --json-error-summary-file? --aggregated-json-errors? --omg-this-is-a-long-switch-name? Whatever floats your boat. The --help that says "implies --verify" seems pretty explicit, but yes, with it's current name it may not be quite as obvious that it triggers verification.

I really just need someone with write privileges to make up their mind, I'll change that code to whatever they want, then that someone can click the big purple button of freedom :)

Reply:
`--json-error-summary-file` seems very descriptive to me

Reply:
That works for me too. Just make sure we have a one character alias for this.